Header header
string[] namespaces
int32[] face_scores
int32 num_robot
bool is_valid
